.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Test Formula
+        run: brew audit --strict --formula --except=version Formula/saucectl.rb
```

Comment:
You might wonder what does `--except=version` do?
Brew parses the version from the URL and the _version_ field. If both fields have a parse-able version, brew's style check complains. We can't omit the version field through go-releaser, which is what creates the formula for us. Hence skipping this style check.